frontend/src/components/question-creation/QuestionElementItem.tsx

```diff
@@ -39,12 +39,20 @@ const renderQuestionContent = (content: QuestionElement) => {
           key={id}
           data={data as MultipleChoiceData}
           id={id}
+          type={QuestionElementType.MULTIPLE_CHOICE}
+        />
+      );
+    case QuestionElementType.MULTI_SELECT:
+      return (
+        <MultipleChoiceElement
+          key={id}
+          data={data as MultipleChoiceData}
```

Comment:
Could you update `MultipleChoiceData` -> `MultiOptionData` or something like that?